pkgs/by-name/ni/nix-search/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "nix-search";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "nix-search";
+    rev = "v${version}";
+    hash = "sha256-dOdcXKfSwi0THOjtgP3O/46SWoUY+T7LL9nGwOXXJfw=";
+  };
+
+  vendorHash = "sha256-bModWDH5Htl5rZthtk/UTw/PXT+LrgyBjsvE6hgIePY=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
```

Comment:
Would be good to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook